usr/lib/cca_stdll/cca_specific.c

```diff
@@ -295,7 +295,7 @@ static CK_RV cca_key_gen(STDLL_TokData_t *tokdata, TEMPLATE *tmpl,
 static CK_RV init_cca_adapter_lock(STDLL_TokData_t *tokdata)
 {
     struct cca_private_data *cca_private = tokdata->private_data;
-    pthread_rwlockattr_t attr = NULL;
+    pthread_rwlockattr_t attr;
```

Comment:
please squash this commit with the commit that added that code (which will remove the change entirely).